src/report_generator.py

```diff
@@ -50,9 +50,9 @@ def generate_daily_report(self, markdown_file_path):
         report = self.llm.generate_daily_report(markdown_content)  # 调用LLM生成报告
 
         report_file_path = os.path.splitext(markdown_file_path)[0] + "_report.md"
-        with open(report_file_path, 'w+') as report_file:
+        with open(report_file_path, 'w+', encoding='utf-8') as report_file:
```

Comment:
This is proved to be working, without the fix when running in windows we get
`Lib\encodings\cp1252.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
UnicodeEncodeError: 'charmap' codec can't encode characters in position 7-10: character maps to <undefined>`